### DISCUSSION TITLE
Contact window on Android

### DISCUSSION THREAD

**Discussion by Mussaka** [tutao/tutanota#4191] [Ideas]
Hi.

I don't know if they are good ideas but I have some suggetion to improve the Contact window on Android.
![Screenshot_20220521-165957_Tutanota-withnotes](https://user-images.githubusercontent.com/93584729/169658789-25732ae8-afd2-42f7-900a-2c502619d7b6.jpg)

Is it possible to have a direct Contact button to replace the default Contact app of my phone ?

I would like to congrat and thank you for your hard work.
Mussaka

**Comment by charlag** [comment]
Hi, thanks for the suggestions, here are my thoughts:
 - filters could be nice but it's not a very common function. Also they can't be where you positioned them, it overlaps the dropdown
 - Phone number is again interesting but is it needed in a list? Do you get useful information from a quick glance at the phone number?
 - Alphabetic bar would be awesome
 - It would be possible to create a direct link to contacts. We also do have an issue about contact integration: #1066

**Reply by Mussaka** [reply]
Hi, thanks for your answer.

- mmhh do you mean that you could integrate a function that other don't have... ;-)
- About the phone number, It would have been better if I explained what's on my mind... I thought of the possibility to tap on the Contact, Email or the Number to directly jump to the Contact sheet, Email writer or Phone dial
- I think so too ;-)
- Thanks for the link, I will take a look.

I would like to say that I am ignorant of the work those change would need.

Thanks again
Mussaka

**Reply by charlag** [reply]
I think it doesn't save too many taps: either you select/unselect filters first and then you click on an item in the list or you just find contact, open it and click on phone number. Need some experimentation whether it is understandable.

It's okay to discuss such things, contact viewer needs some love.
